Vagrantfile

```diff
@@ -201,7 +201,7 @@ Vagrant.configure(2) do |config|
     # Che is active once it returns success        
     while [ true ]; do
       printf "#"
-      curl -v ${CHE_URL}/dashboard &>/dev/null
```

Comment:
You are right. This check only verify if host is responding or not. I've changed it to:  
`curl -I http://"${CHE_HOST_IP}":"${CHE_PORT}"/api/ 2>/dev/null | head -n 1 | cut -d' ' -f2`